docs/DOCKER.rst

```diff
@@ -47,19 +39,28 @@ For running an S3 AWS backend, you will have to add a new section
             "credentialsProfile": "default"
         }
     }
+(...)
 
-You will also have to mount your AWS credentials file:
+You will also have to edit your AWS credentials file to be able to use your
+command line tool of choice. This file should only state credentials for S3 AWS
+hosted data, not for your Zenko CloudServer hosted data.
+As for your locationConfig.json, you will need to mount your AWS credentials
+file at run time:
 ``-v ~/.aws/credentials:/root/.aws/credentials`` on Linux, OS X, or Unix or
 ``-v C:\Users\USERNAME\.aws\credential:/root/.aws/credentials`` on Windows
 
-.. code:: shell
+S3DATA=file
```

Comment:
I think we should simplify this and only mention S3DATA=multiple. If they want to do a quickstart with S3BACKEND=file or S3BACKEND=mem then it's all file or mem, respectively.  If they want a file or mem data location, they can do that through the locationConfig.json.

Reply:
As I mentioned below, a lot of people are using S3BACKEND=mem, so I believe it's worth mentioning it here.
Regarding quickstart, this section is not under the "Getting Started" section, but is its own section, so I think it should provide a bit more detail and config options about using CloudServer in Docker. I agree that the "Getting Started" section should be simplified, though.

Reply:
We should not mention S3DATA=mem.  As I said, I'm fine with S3BACKEND=mem.

Reply:
Changed, thanks